PR: Add Quickstart page containing an interactive tour of Spyder's UI (#171)
spyder-ide/spyder-docs: doc/quickstart.rst

##########
Quickstart
##########

.. raw:: html

   <div class="interactive-tour-container">
      <img id="tour-default-layout-main" class="tour-screenshot tour-screenshot-active" src="_static/tour/default-layout-main.png">
      <img id="tour-options-menu" class="tour-screenshot" src="_static/tour/options-menu.png">
      <img id="tour-context-menu" class="tour-screenshot" src="_static/tour/context-menu.png">
      <img id="tour-variable-explorer" class="tour-screenshot" src="_static/tour/variable-explorer.png">
      <img id="tour-plots" class="tour-screenshot" src="_static/tour/plots.png">
      <img id="tour-files" class="tour-screenshot" src="_static/tour/files.png">
      <img id="tour-find" class="tour-screenshot" src="_static/tour/find.png">
      <img id="tour-profiler" class="tour-screenshot" src="_static/tour/profiler.png">

>       <img id="tour-code-analysis" class="tour-screenshot" src="_static/tour/code-analysis.png">

Author: This code is correct, but the image files names are swapped (pr`profiler.png` is the code analysis and vice versa), and so in the tour itself they are reversed.

Author: Fixed, thanks!

      
      <svg class="tour-overlay" viewBox="0 0 1280 720">
        <rect id="introduction-rect" class="highlight-rect tour-default-layout-main" x="5" y="5" width="1270" height="710" />

>         <rect id="toolbar-rect" class="highlight-rect tour-default-layout-main" x="5" y="5" width="1270" height="32"" />

Author: ```suggestion
        <rect id="toolbar-rect" class="highlight-rect tour-default-layout-main" x="5" y="5" width="1270" height="32" />
```

##########
Quickstart
##########

Welcome to our Quickstart!
Here you will find an interactive tour that will guide you through Spyder's interface. 

> You will get familiar with the most important parts of the IDE, especially those we will be mentioning throughout our docs. 
> Finally, you will get to walk through some of Spyder's key panes and functionality.

Reviewer: ```suggestion
You'll get familiar with the most important parts of the IDE, especially those we'll be mentioning throughout our docs. 
Finally, you'll get to walk through some of Spyder's key panes and functionality.
```

Too many `will`s make things too formal, in my opinion.

Author: That's how I would write it for a native English speaker, but per the style guide and our previous discussions and PRs, contractions should be strongly avoided due to the increase in difficulty for non-native speakers. So I'm not sure what we should do here.

Reviewer: I think in this case is fine to use contractions to make text a bit friendlier. But let's see @juanis2112 thinks.

Quickstart
##########

> Welcome to our Quickstart! Here you will find a tour that will guide you through Spyder's interface. 

Author: ```suggestion
Welcome to our Quickstart!
Here you will find an interactive tour that will guide you through Spyder's interface. 
```

Added line break and "interactive"

> You will get familiar with some of the elements found in the IDE, especially those that we will be mentioning through all our docs. 

Author: ```suggestion
You will get familiar with the most important parts of the IDE, especially those we will be mentioning throughout our docs. 
```

Use more compelling language, and clearer wording

> Finally, you will walk through some of the panes and its main functionality.

Author: ```suggestion
Finally, you will get to walk through some of Spyder's key panes and functionality.
```

Fix grammar and clearer, simpler phrasing
